src/test/java/de/retest/recheck/persistence/xml/XmlTransformerTest.java

```diff
@@ -0,0 +1,46 @@
+package de.retest.recheck.persistence.xml;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.jupiter.api.Test;
+
+import de.retest.recheck.persistence.xml.util.StdXmlClassesProvider;
+import de.retest.recheck.ui.descriptors.Element;
+import de.retest.recheck.ui.descriptors.SutState;
+
+class XmlTransformerTest {
+
+	@Test
+	void should_not_mix_data_from_multiple_xml_files() throws Exception {
+		final String base = "src/test/resources/de/retest/recheck/persistence/xml/";
```

Comment:
Looks like a constant value, why it isn't?

Reply:
It is only value for this particular test method, not the entire test class, so why should I increase the scope if this thing if not necessary? However, since there are no other tests, I can introduce a constant.